### DISCUSSION TITLE
Is there a script to start all Connections services (including Kubernetes and Component pack)

### DISCUSSION THREAD

**Discussion by arnaldovale** [HCL-TECH-SOFTWARE/connections-automation#180] [Q&A] [answered]
After everything is installed and all servers are rebooted, is there a script to start all Connections services (including Kubernetes and Component pack)?
To be more specific, which commands are necessary to start the component pack?

**Comment by sabrina-yee** [comment, marked as answer]
The pods have restartPolicy of `Always` so they should start automatically.

**Reply by stoeps13** [reply]
The pods, haproxy and nginx are starting automatically, but IHS (HTTP) and WebSphere not.

**Reply by sabrina-yee** [reply]
> The pods, haproxy and nginx are starting automatically, but IHS (HTTP) and WebSphere not.

That's right.  For IHS and WebSphere there is a [restart playbook](https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/playbooks/third_party/was-nd-restart.yml) that can model from to create a start playbook.
